src/NUnitEngine/nunit.engine/Internal/TargetFrameworkHelper.cs

```diff
@@ -92,17 +92,6 @@ public string FrameworkName
                     break;
                 }
 
-                foreach (var reference in _module.AssemblyReferences)
-                    if (reference.Name == "mscorlib" &&
-                        BitConverter.ToUInt64(reference.PublicKeyToken, 0) == 0xac22333d05b89d96)
-                    {
-                        // We assume 3.5, since that's all we are supporting
-                        // Could be extended to other versions if necessary
-                        // Format for FrameworkName is invented - it is not
-                        // known if any compilers supporting CF use the attribute
-                        return ".NETCompactFramework,Version=3.5";
-                    }
-
```

Comment:
We'd previously kept this code, for the purpose of providing a sensible error message if someone tried to run .NET CF 3.5 tests in the console. I'm not strongly against removing it, but it doesn't seem to be causing problems to keep it.